app/src/features/surveys/CreateSurveyPage.tsx

```diff
@@ -182,6 +184,24 @@ const CreateSurveyPage = () => {
     .concat(ProprietaryDataYupSchema)
     .concat(AgreementsYupSchema);
 
+  useEffect(() => {
+    const getPermitNumbers = async (projectId: number) => {
+      const permitNumbersResponse = await biohubApi.survey.getSurveyPermitNumbers(projectId);
```

Comment:
Would it make sense to bundle this code in the existing `getProject` function/useEffect?